tools/vcli/bw/repl/repl.go

```diff
@@ -88,10 +89,50 @@ func SimpleReadLine(done chan bool) <-chan string {
 	return c
 }
 
+// startProfilingIfEnabled starts profiling for the current call to runBQLFromFile if profilingEnabled is true,
+// returning the two files to which the profiling metrics will be printed.
+func startProfilingIfEnabled(profilingEnabled bool) (*os.File, *os.File) {
```

Comment:
I am not sure if it would be better to have a function startProfiling()
and check if it is enabled elsewhere before calling startProfiling

Reply:
I do not like this `IfEnabled` as well, when I first wrote this I also had mixed feelings about it. That is why in a later commit I change this to be just a `startProfiling` function, and check if it is enabled elsewhere (out of the scope of the function).